deployments/cloudfoundry/bosh/DEVELOPMENT.md

```diff
@@ -6,19 +6,40 @@
 
 [Quick Start Guide](https://bosh.io/docs/bosh-lite/)
 
-### Supplementary Getting Started Checklist
+### Start the Bosh Director
 
-- Make sure director is running
-    - Quick start guide to run a BOSH Director can be found [here.](https://bosh.io/docs/quick-start/)
-- Need to upload right Ubuntu/OS blob
-    - Guide can be found [here.](https://bosh.io/docs/uploading-stemcells/)
-    - The following is an example command to upload the Warden (BOSH Lite) Ubuntu Bionic (18.04.6 LTS) stemcell:
+Quick start guide to run a BOSH Director can be found [here.](https://bosh.io/docs/quick-start/)
+
+Debugging for Bosh Director startup issues
```

Comment:
should this be a header?

Reply:
Fixed formatting here, hopefully it's more clear now.